prow/prstatus/prstatus.go

```diff
@@ -86,6 +83,14 @@ type Label struct {
 	Name githubql.String
 }
 
+// Context holds graphql response data for github contexts.
+type Context struct {
```

Comment:
Why use the `githubql` types if we have to coerce into them anyway?

Reply:
Fixed